.github/workflows/closing.yml

```diff
@@ -5,8 +5,14 @@ on:
   issues:
     types: [closed]
 
+permissions:
+  contents: read
+
```

Comment:
This file is maintained in https://github.com/WeblateOrg/meta/, any changes in it will be overwritten on next sync. Please do changes there.